src/styles/blocks/calendar.scss

```diff
@@ -1,20 +1,23 @@
 .calendar {
   display: flex;
 
-  width: $calendarRow * $size + 2 * $calendarPadding
+  width: $calendarRow * $size + 1
     + ($calendarRow - 1) * $gap;
 
-  box-sizing: border-box;
+  gap: 1px;
+
+  margin-left: auto;
+  margin-right: auto;
+
```

Comment:
you also should center it vertically, I would suggest adding a container for the whole calendar to make it easier

Reply:
"[STYLES] - Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover."

This makes me believe that I shouldn't add any height to my calendar (except paddings if I added hover transform).
And since everything is a border-box, adding height seems counter productive.